cache/in-memory/src/updates.rs

```diff
@@ -245,45 +245,35 @@ impl UpdateCache for GuildUpdate {
             return;
         }
 
-        let mut guild = match cache
-            .0
-            .guilds
-            .get_mut(&self.0.id)
-            .map(|r| Arc::clone(r.value()))
-        {
-            Some(guild) => guild,
-            None => return,
+        if let Some(mut guild) = cache.0.guilds.get_mut(&self.0.id) {
```

Comment:
This can be ?'d away to avoid a level and `if let`

Reply:
the update method does not return a result though

Reply:
right